esp-hal-common/src/soc/esp32s3/ulp_core.rs

```diff
@@ -1,4 +1,39 @@
 //! Control the ULP RISCV core
+//!
+//! ## Overview
+//!
+//! The `ULP RISCV CORE` peripheral allows control over the `Ultra-Low Power
```

Comment:
If you're touching this, could you please globally update RISCV to RISC-V?

Reply:
Done